tests/Feature/Empleados/EmployeeScheduleAssignmentTest.php

```
@@ -90,6 +94,129 @@
+        $company->id,
+        fn () => new PayrollContextTargets(
+            profileIds: [$profile->id],
+            employeeIds: [$employee->id],
+        ),
+        function ($context) use (&$profileContext): void {
+            $profileContext = $context;
+        },
+        function ($context) use (&$profileContext, &$publicationContext): void {
+            $publicationContext = $context;
+
+            expect(fn () => $profileContext->assertStage(
```

PHPStan (2.2.7)

[error] Cannot call method assertStage() on null.


(method.nonObject)